core-java-modules/core-java-collections-list-6/src/main/java/com/baeldung/enumwithallhttpstatuscodes/HTTPStatus.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.enumwithallhttpstatuscodes;
+
+public enum HTTPStatus {

```

Comment:
Also note that we follow the https://google.github.io/styleguide/javaguide.html and so HTTP should be written as Http